config/initializers/hesa_ethnicities.rb

```diff
@@ -19,29 +19,34 @@ module HesaEthnicityValues
   INFORMATION_REFUSED = 'Information refused'.freeze
 end
 
-# https://www.hesa.ac.uk/collection/c19053/e/ethnic
-HESA_ETHNICITIES_2019_2020 = [
-  ['10', HesaEthnicityValues::WHITE],
-  ['15', HesaEthnicityValues::GYPSY_OR_TRAVELLER],
-  ['21', HesaEthnicityValues::CARIBBEAN],
-  ['22', HesaEthnicityValues::AFRICAN],
-  ['29', HesaEthnicityValues::OTHER_BLACK],
-  ['31', HesaEthnicityValues::INDIAN],
-  ['32', HesaEthnicityValues::PAKISTANI],
-  ['33', HesaEthnicityValues::BANGLADESHI],
-  ['34', HesaEthnicityValues::CHINESE],
-  ['39', HesaEthnicityValues::OTHER_ASIAN],
-  ['41', HesaEthnicityValues::WHITE_AND_BLACK_CARIBBEAN],
-  ['42', HesaEthnicityValues::WHITE_AND_BLACK_AFRICAN],
-  ['43', HesaEthnicityValues::WHITE_AND_ASIAN],
-  ['49', HesaEthnicityValues::OTHER_MIXED],
-  ['50', HesaEthnicityValues::ARAB],
-  ['80', HesaEthnicityValues::OTHER_ETHNIC],
-  ['90', HesaEthnicityValues::NOT_KNOWN],
-  ['98', HesaEthnicityValues::INFORMATION_REFUSED],
-].freeze
+module HesaEthnicityCollections
+  # https://www.hesa.ac.uk/collection/c19053/e/ethnic
+  HESA_ETHNICITIES_2019_2020 = [
+    ['10', HesaEthnicityValues::WHITE],
+    ['15', HesaEthnicityValues::GYPSY_OR_TRAVELLER],
+    ['21', HesaEthnicityValues::CARIBBEAN],
+    ['22', HesaEthnicityValues::AFRICAN],
+    ['29', HesaEthnicityValues::OTHER_BLACK],
+    ['31', HesaEthnicityValues::INDIAN],
+    ['32', HesaEthnicityValues::PAKISTANI],
+    ['33', HesaEthnicityValues::BANGLADESHI],
+    ['34', HesaEthnicityValues::CHINESE],
+    ['39', HesaEthnicityValues::OTHER_ASIAN],
+    ['41', HesaEthnicityValues::WHITE_AND_BLACK_CARIBBEAN],
+    ['42', HesaEthnicityValues::WHITE_AND_BLACK_AFRICAN],
+    ['43', HesaEthnicityValues::WHITE_AND_ASIAN],
+    ['49', HesaEthnicityValues::OTHER_MIXED],
+    ['50', HesaEthnicityValues::ARAB],
+    ['80', HesaEthnicityValues::OTHER_ETHNIC],
+    ['90', HesaEthnicityValues::NOT_KNOWN],
+    ['98', HesaEthnicityValues::INFORMATION_REFUSED],
+  ].freeze
 
-# Two codes have been dropped in 2020/21
-# https://www.hesa.ac.uk/collection/c20053/e/ethnic
-HESA_ETHNICITIES_2020_2021 = HESA_ETHNICITIES_2019_2020
-                               .reject { |ethnicity| %w[90 98].include?(ethnicity.first) }
+  # Two codes have been dropped in 2020/21
```

Comment:
Just double checking this claim, it looks like `90` and `98` are still present in all of the docs (2020 though to 2022)

Reply:
@TobyRet can you shed any light on this? The HESA data sets seem to contain these two codes for all revisions I can find.

Reply:
@benswannack having chatted with Toby we're agreed the documentation no longer omits these values, not sure why but I've echoed this in https://github.com/DFE-Digital/apply-for-teacher-training/pull/5524/commits/9a00debec76d6c5384e290adcd313fb6c6e21325